PowerShell/ScubaGear/Modules/Providers/ExportAADProvider.psm1

```diff
@@ -274,6 +291,9 @@ function Get-PrivilegedUser {
                 if ($Objecttype -eq "user") {
                     LoadObjectDataIntoPrivilegedUserHashtable -RoleName $Role.DisplayName -PrivilegedUsers $PrivilegedUsers -ObjectId $User.Id -TenantHasPremiumLicense $TenantHasPremiumLicense -M365Environment $M365Environment -Objecttype "user"
                 }
+                elseif ($Objecttype -eq "servicePrincipal") {
+                    LoadObjectDataIntoPrivilegedUserHashtable -RoleName $Role.DisplayName -PrivilegedUsers $PrivilegedUsers -ObjectId $User.Id -TenantHasPremiumLicense $TenantHasPremiumLicense -M365Environment $M365Environment -Objecttype "serviceprincipal"
```

Comment:
Can we add a new unit test to handle $Objecttype="serviceprincipal" in [LoadObjectDataIntoPrivilegedUserHashTable.Tests.ps1](https://github.com/cisagov/ScubaGear/blob/main/PowerShell/ScubaGear/Testing/Unit/PowerShell/Providers/AADProvider/LoadObjectDataIntoPrivilegedUserHashtable.Tests.ps1)?